### PR TITLE
Switch to Remora.Discord

### PR DESCRIPTION
**Pull request by Octol1ttle**
result checks go brrr

this also involves switching to using Discord's modern stuff like embeds and interactions

and using brand-new for me programming concepts (dependency injection, results)

### CODE REVIEW

File: Commands/BanCommand.cs [hunk 1]
```diff
@@ -147,7 +147,15 @@ var interactionResult
...
 
-        //TODO: Проверка на существующий бан.
+        var existingBanResult = await _guildApi.GetGuildBanAsync(guildId.Value, target.ID, CancellationToken);
+        if (!existingBanResult.IsDefined()) {
+            return (Result)await _feedbackService.SendContextualEmbedAsync(
+                new Embed(
+                          Title: Messages.UserNotBanned,
+                          Colour: ColorsList.Red
+                    ),
+                ct: CancellationToken);
```
Comment (Octol1ttle) [inline Commands/BanCommand.cs:157]: Please use EmbedBuilders instead, and add the current user avatar to the embed

File: Messages.ru.resx [hunk 2]
```diff
@@ -492,4 +492,7 @@
     <data name="UserAlreadyBanned" xml:space="preserve">
         <value>Этот пользователь уже забанен!</value>
     </data>
+    <data name="UserUnbanned" xml:space="preserve">
+        <value>{0} был(-а) забанен(-а)</value>
```
Comment (Octol1ttle) [inline Messages.ru.resx:496]: You forgot to update this

File: Commands/BanCommand.cs (hunk 1 above)
Comment (neroduckale) [reply Commands/BanCommand.cs:157]: Ok, fixes coming soon

Comment (neroduckale) [reply Commands/BanCommand.cs:157]: Ok, fixes coming soon

**Comment by Octol1ttle** [comment]
@mctaylors Please try testing your work on adding the `/mute` command again. If everything works, push directly to `remora` on this repo, hmu on IRC or Matrix if help is needed

**Review by mctaylors — DISMISSED** [review]

**Review by mctaylors — APPROVED** [review]

**Review by neroduckale — APPROVED** [review]

**Pull request merged**